src/Processors/Streaming/ReplayStreamTransform.cpp

```diff
@@ -16,19 +16,29 @@ namespace Streaming
 
 constexpr Int64 MAX_WAIT_INTERVAL_MS = 500;
 
-ReplayStreamTransform::ReplayStreamTransform(const Block & header, Float32 replay_speed_)
-    : ISimpleTransform(header, header, true, ProcessorID::ReplayStreamTransformID), replay_speed(replay_speed_)
+ReplayStreamTransform::ReplayStreamTransform(const Block & header, Float32 replay_speed_, const Int64 & sahrd_last_sn_)
+    : ISimpleTransform(header, header, true, ProcessorID::ReplayStreamTransformID)
+    , replay_speed(replay_speed_)
+    , shard_last_sn(sahrd_last_sn_)
```

Comment:
A typo `sahrd_last_sn_`

Reply:
We should validate the `shard_last_sn`, when `< 0`, it's invalid sequence number, set `enable_display = false`

Reply:
Is it possible the `shard_last_sn` < 0